data_structures/queue/fifo_animal_shelter.py

```diff
@@ -0,0 +1,60 @@
+from node import Node
+
+
+class AnimalShelter:
+    """Create a FIFO AnimalShelter class"""
+    def __init__(self, iter=[]):
+        self.front = None
+        self.back = None
+        self._size = 0
+
+        for item in iter:
+            self.enqueue(item)
+
+    def __repr__(self):
+        """Returns the animal at the front of the queue."""
+        return f'The animal in the shelter longest is: {self.front.val}'
+
+    def enqueue(self, animal=None):
```

Comment:
`animal` should not be a optional argument, thus you should not be setting a default for it.